src/libraries/Microsoft.Extensions.Caching.Memory/src/MemoryCache.cs

```diff
@@ -209,23 +197,24 @@ public bool TryGetValue(object key, out object result)
             ValidateCacheKey(key);
             CheckDisposed();
 
-            DateTimeOffset utcNow = _options.Clock.UtcNow;
+            DateTime utcNow = UtcNow;
 
             CoherentState coherentState = _coherentState; // Clear() can update the reference in the meantime
-            if (coherentState._entries.TryGetValue(key, out CacheEntry entry))
+            if (coherentState._entries.TryGetValue(key, out CacheEntry tmp))
             {
+                CacheEntry entry = tmp;
```

Comment:
what was your motivation behind this change? What do we gain by having two local variables instead of one?

Reply:
The out variable must live on stack, the other local variable can be enregistered.